src/main/java/com/artipie/rpm/Repomd.java

```diff
@@ -260,6 +198,31 @@ private static Completable gzip(final Path input, final Path output) {
             });
     }
 
+    /**
+     * Result from storage.
+     *
+     * @param nodes XML node
+     * @param file Path of file
+     * @param rxsto RXTO wrapper
+     * @return Result from storage
+     */
+    private static Completable resultFromStorage(
+        final List<XML> nodes,
+        final Path file,
+        final RxStorageWrapper rxsto) {
```

Comment:
@Guseyn this class can be instantiated internally, since it's just a wrapper for object's state `storage` field. We can use non-static method and construct it right in the body. Method parameter looks redundant here.

Reply:
@g4s8 fixed.